itest/rust/build.rs

```diff
@@ -424,8 +424,8 @@ fn generate_property_template(inputs: &[Input]) -> PropertyTests {
             export_multiline: GString,
             #[export(range = (0.0, 20.0))]
             export_range_float_0_20: f64,
-            // We're missing step currently.
-            //  #[export(range = (-10, 20 /*, 0.2 */))] export_range_float_neg_10_20_02: f64,
+             #[export(range = (-10.0, 20.0, 0.2))]
+            export_range_float_neg_10_20_02: f64,
```

Comment:
Indentation and `neg` spacing:
```suggestion
            #[export(range = (-10.0, 20.0, 0.2))]
            export_range_float_neg10_20_02: f64,
```